crate_universe/private/crate.bzl

```diff
@@ -114,6 +115,8 @@ def _annotation(
         data (list, optional): A list of labels to add to a crate's `rust_library::data` attribute.
         data_glob (list, optional): A list of glob patterns to add to a crate's `rust_library::data` attribute.
         deps (list, optional): A list of labels to add to a crate's `rust_library::deps` attribute.
+        gen_binaries (list or bool, optional): As a list, the subset of the crate's bins that should get `rust_binary`
```

Comment:
Could you also make this configurable globally like `gen_build_script` is?

Reply:
Done.

Reply:
Could you also add a note about this not affecting binary only crates? Maybe a section could be added to the top level `crate_universe` docs in `//crate_universe:docs.bzl`? Not sure where the best place is for this info but I can see that being a point of confusion for folks (already was but I want to try and avoid a bug report being incorrectly filed for this new feature)